### PR TITLE
Contrail images

### PR DESCRIPTION
**Pull request by eonpatapon**

### CODE REVIEW

File: pkgs/docker-images/default.nix [hunk 1]
```diff
@@ -3,35 +3,40 @@
 let
   configuration = import ./configuration.nix pkgs;
 
-  buildContrailImageWithPerp = name: command: preStartScript:
+  buildContrailImageWithPerp = name: command: preStart:
```
Comment (nlewo) [inline pkgs/docker-images/default.nix]: Please use preStartScript variable name.

File: pkgs/docker-images/default.nix [hunk 2]
```diff
@@ -3,35 +3,40 @@
...
+      '' + preStart;
       extraCommands = "chmod u+w etc; mkdir -p var/log/contrail etc/contrail";
   };
 
 in
 {
   contrailApi = buildContrailImageWithPerp "opencontrail/api"
     "${contrail32Cw.api}/bin/contrail-api --conf_file /etc/contrail/contrail-api.conf"
-    ''consul-template-wrapper -- -once  -template="${configuration.api}:/etc/contrail/contrail-api.conf"'';
+    ''consul-template-wrapper -- -once  -template="${configuration.contrailApi}:/etc/contrail/contrail-api.conf"'';
```
Comment (nlewo) [inline pkgs/docker-images/default.nix]: This changes seems to not be related to this PR.

Comment (eonpatapon) [reply pkgs/docker-images/default.nix]: Yes it is

File: pkgs/docker-images/default.nix (hunk 1 above)
Comment (eonpatapon) [reply pkgs/docker-images/default.nix]: Done

File: pkgs/docker-images/default.nix (hunk 2 above)
Comment (nlewo) [reply pkgs/docker-images/default.nix]: Ok, my bad :/

**Pull request merged**